### DISCUSSION TITLE
Best way to get renderer info?

### DISCUSSION THREAD

**Discussion by alexpineda** [pmndrs/postprocessing#405] [Q&A] [answered]
If I console log the renderer I pass in I only get this.
![image](https://user-images.githubusercontent.com/586716/188077415-67f7282e-e9f1-42e7-a5dd-5e518478e759.png)

I've seen some nice renderer info overlays maybe that could help me. I'm just not sure where to find them and whether it'll take care of this little issue.

**Comment by vanruesc** [comment, marked as answer]
Do this:

```js
renderer.info.autoReset = false;
```

and then in your render loop before calling `composer.render`:

```js
composer.getRenderer().info.reset();
```
